### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by why-a-en**
In this pull request,
- I created an array with nested objects that stores all of the information of each project card
- The project section is created dynamically using the information stored in the objects
- All of the HTML in project-section is created when the page loads
- I created pop-up window that looks good on both mobile and desktop screen

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -300,6 +301,132 @@ main {
...
+  width: 45%;
+}
+
+#seeSource {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+  width: 50%;
+}
```
Comment (christianbiring1) [inline styles.css:412]: - [ ] Kindly make sure you stick to the template as much as possible. the color, spacing(margin, padding) should be similar to the template in Figma. As you may see the bg-color of your button and the one in figma wich are quite different.
- [ ] Also consider having the same spacing(padding, margin) all around the content of the pop up as is the case on the original design.

### Your design :point_down:

Screenshot from 2023-03-09 16-11-32

### Figma template :point_down: 

image

File: script.js [hunk 2]
```diff
@@ -14,3 +15,228 @@ navItems.forEach((navItem) => navItem.addEventListener('click', () => {
...
+                        </div>
+                      </header>
+                      <div class="tech-box popup">
+                        <ul>
+                          <li>${cardsInfo[0].tech_stack.tech0}</li>
+                          <li>${cardsInfo[0].tech_stack.tech1}</li>
+                          <li>${cardsInfo[0].tech_stack.tech2}</li>
+                          <li>${cardsInfo[0].tech_stack.tech3}</li>
+                        </ul>
+                      </div>
```
Comment (christianbiring1) [inline script.js:201]: - [ ] As specified in the project requirements, Kindly make sure When the user clicks (or taps) the button to check project details, `the popup with details about the project appears`. As you may notice your modals are showing all the same exact data except the titles, whereas even the `details`, `pictures` should suppose to match with the project the user clicks at. Kindly consider changing that.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @anbehindY  @JansueT , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
almost

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] The project's data are populated dynamically :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @anbehindY , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: dynamic popups

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**